dense_vector/track.py

```diff
@@ -0,0 +1,31 @@
+import random
+
+
+def knn_param_source(track, params, **kwargs):
+    # choose a suitable index: if there is only one defined for this track
+    # choose that one, but let the user always override index
+    random.seed(31)
+    if len(track.indices) == 1:
+        default_index = track.indices[0].name
+    else:
+        default_index = "_all"
+
+    index_name = params.get("index", default_index)
+
+    return {
+        "body": {
+            "knn": {
+                "field": "vector",
+                "query_vector": params.get("query-vector", [random.uniform(-1.0, 1.0) for _ in range(96)]),
```

Comment:
Just to clarify, we will still have an option to provide a specific `query-vector`?
One thing we wanted to add to the track is to calculate recall metric, and this depends on the query to be specific.

Reply:
@mayya-sharipova that's right, you're still able to specify a specific query vector. It only randomly generates one if the `query-vector` parameter is missing. The current PR still uses specific queries, and doesn't use randomization yet.